config.yml.mustache

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+
+jobs:
+  build:
+    parameters:
+      coq:
+        type: string
+    docker:
+    - image: coqorg/coq:<<parameters.coq>>
+    resource_class: medium
+    environment:
+      OPAMJOBS: 2
+      OPAMVERBOSE: 1
+      OPAMYES: true
+      TERM: xterm
+    steps:
+    - checkout
+    - run:
+        name: Pull submodules
+        command: git submodule update --init --remote
+    - run:
+        name: Configure environment
+        command: echo . ~/.profile >> $BASH_ENV
+    - run:
+        name: Install dependencies
+        command: |
+          opam repo -a --set-default add coq-extra-dev https://coq.inria.fr/opam/extra-dev
+          opam update
+          opam install --deps-only .
+    - run:
+        name: List installed packages
+        command: opam list
+    - run:
+        name: Build, test, and install package
+        command: opam install -t .
+{{# ci_test_dependants }}
+    - run:
+        name: Test dependants
+        command: |
+          PINS=$(echo `opam list -s --pinned --columns=package` | sed 's/ /,/g')
```

Comment:
The <code>$(echo \`opam list…\` | sed 's/ /,/g')</code> phrasing is OK, but maybe a bit hacky ;)
maybe we could have a sed-free solution with some array + IFS?

```
PINS=($(opam list -s --pinned --columns=package)); PINS=$(IFS=, ; echo "${PINS[*]}")
```
but this requires bash (inspired by https://stackoverflow.com/a/9429887/9164010)

Reply:
Or maybe this is even simpler (no sed, only one subshell):

```
PINS=$(opam list -s --pinned --columns=package | xargs | tr ' ' ,)
```